### PR TITLE
Date update

### PR DESCRIPTION
**Pull request by denis-yuen**
n/t

### CODE REVIEW

File: _includes/submit.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Submissions
 
-Abstract submissions will open in March 2020.
+Abstract submissions will open on April 30, 2020.
```
Comment (nlharris) [inline _includes/submit.md]: Abstract submission will CLOSE on April 30, 2020.

**Review by nlharris — CHANGES_REQUESTED** [review]
Thanks, looks good except for the correction I noted on the submit page.

**Review by tnabtaf — APPROVED** [review]

**Review by morgantaschuk — APPROVED** [review]

**Review by nlharris — APPROVED** [review]
looks good, thanks. check with @tnabtaf whether we should change the date it says registration will open, if it doesn't seem like we'll be able to open it tomorrow.

**Comment by nlharris** [comment]
"Registration opens" needs to be changed to "Early April".

**Comment by denis-yuen** [comment]
(seems superseded by other PR)

**Comment by nlharris** [comment]
sorry, denis--i hadn't seen that you made that change!

**Comment by denis-yuen** [comment]
No worries!

**Pull request closed without merging**